test-requirements.txt

```diff
@@ -0,0 +1,5 @@
+django>=1.4, <1.5
+-r requirements.txt
+-e git+https://github.com/edx/xblock-sdk.git@629650ed6fc1b77d1a5287debeace0b29db7c0fd#egg=xblock-sdk
```

Comment:
@singingwolfboy Without the `-e` flag it doesn't seem possible to install test requirements for xblock-sdk (`pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/base.txt` and `pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/test.txt` in `travis.yml` fail with a "No such file or directory" error), so I'm keeping it for now.